### PR TITLE
feat(CI): build the final container using podman as well

### PR DESCRIPTION
**Pull request by skateman**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -47,7 +47,11 @@ jobs:
...
       uses: actions/checkout@v2
     - name: Set up the environment
-      run: cp .env.example .env
+      run: |
+        cp .env.example .env
+        pip3 install podman-compose
+    - name: Build using podman-compose
+      run: podman-compose build
     - name: Build using docker-compose
       run: docker-compose build
```
Comment (Victoremepunto) [inline .github/workflows/ci.yml:56]:
```suggestion
```

Comment (skateman) [reply .github/workflows/ci.yml:56]: What's the difference? :eyes:

Comment (Victoremepunto) [reply .github/workflows/ci.yml:56]: I thought the purpose of this PR is to replace the docker-compose with podman-compose, the suggestion is to remove the `docker-compose` part :)

Comment (Victoremepunto) [reply .github/workflows/ci.yml:56]: if you want to ensure it works both ways - then ignore me

Comment (skateman) [reply .github/workflows/ci.yml:56]: Yeah, @marleystipich2 is using docker still, so we're making sure both developer setups work.

**Review by romanblanco — APPROVED** [review]
LGTM :+1:

**Pull request merged**